src/components/com_tjcertificate/site/views/certificate/view.html.php

```diff
@@ -75,6 +77,13 @@ public function display($tpl = null)
 		$model = TJCERT::model('Certificate', array('ignore_request' => true));
 		$this->contentHtml = $model->getCertificateProviderInfo($this->certificate->getClientId(), $this->certificate->getClient());
 
+		if ($this->certificate->getClient() === 'com_tjlms.course')
+		{
+			JLoader::import('components.com_tjlms.models.course', JPATH_SITE);
+			$courseModel = JModelLegacy::getInstance('Course', 'TjlmsModel', array('ignore_request' => true));
+			$this->item = $courseModel->getItem($this->certificate->getClientId());
```

Comment:
Call a plugin trigger here, pass context/client and other details and the respective client plugin will send the data if that plugin is available

Reply:
done